pandas/tests/indexes/timedeltas/test_insert.py

```diff
@@ -96,6 +96,11 @@ def test_insert_empty(self):
         idx = timedelta_range("1 Day", periods=3)
         td = idx[0]
 
-        idx[:0].insert(0, td)
-        idx[:0].insert(1, td)
-        idx[:0].insert(-1, td)
+        result = idx[:0].insert(0, td)
```

Comment:
Worth parameterizing?

Reply:
i dont think so.  more interested in trying to parametrize across DTI/TDI, but thats out of scope